addons/sale_stock/tests/test_create_perf.py

```diff
@@ -59,7 +63,7 @@ def test_empty_sales_orders_batch_creation_perf(self):
     def test_dummy_sales_orders_batch_creation_perf(self):
         """ Dummy SOlines (notes/sections) should not add any custom queries other than their insert"""
         # + 4 SOL insert
-        with self.assertQueryCount(admin=40):
+        with self.assertQueryCount(admin=43):
```

Comment:
3 from above

Reply:
@Feyensv I'm not sure why there are only 2 queries more here where intuitively I would say that the comment is right

Reply:
Comment is probably not right anymore since records are INSERT'ed in batch since https://github.com/odoo/odoo/pull/80961.